app/scripts/components/common/card/styles.tsx

```diff
@@ -0,0 +1,161 @@
+import styled from 'styled-components';
+import {
+  glsp,
+  listReset,
+  media,
+  multiply,
+  themeVal,
+  visuallyHidden
+} from '@devseed-ui/theme-provider';
+import { Overline } from '@devseed-ui/typography';
+import { VerticalDivider } from '@devseed-ui/toolbar';
+import { variableGlsp } from '$styles/variable-utils';
+import { VarHeading } from '$styles/variable-components';
+
+export const CardBlank = styled.article`
+  position: relative;
+  display: flex;
+  flex-flow: column nowrap;
+  border-radius: ${multiply(themeVal('shape.rounded'), 2)};
+  box-shadow: ${themeVal('boxShadow.elevationD')};
+  height: 100%;
+  overflow: hidden;
+  transition: all 0.24s ease-in-out 0s;
+`;
+
+export const CardList = styled.ol`
```

Comment:
`CardList` is being used by instances (ex. GHG : https://github.com/US-GHG-Center/veda-config-ghg/blob/89ec4506cd3a172dfbb6dfa6ffbde8662b448fe3/overrides/home/keypoints.tsx#L7) - We should check other instances too - We should either 1. plan to bring the changes to instances or 2. make this backward compatible.

Reply:
Duplicated those two back to `/common/card/index.tsx` to support instance dependency. Once we update the instances to point to a version of veda with these new updates, we should point the imports to the new location and then remove it from this file.

Reply:
ahh I see and remember what happened now, I changed the original `CardList` name to `CardListGrid` and moved to to `/styles` this is what the ghg instance depended on. Looks like we moved this style dependency up to GHG which is good